### ISSUE TITLE
Wrong calculation of conditional expression loss?

### ISSUE THREAD

**Issue by aiueogawa** [albertpumarola/GANimation#74]
@albertpumarola 
In this implementation, conditional expression loss is calculated as mean squared error. 
The corresponding line is here https://github.com/albertpumarola/GANimation/blob/master/models/ganimation.py#L280.

`torch.nn.MSELoss`, however, actually calculate mean squared error but you divide it by the batch size again.
Is this a simple bug or does it have some meaning?

**Comment by albertpumarola** [comment]
Batch size is part of the lambda. This config is the one that work the best for me but I did not have many resources to test multiple hyperparameters.
